nuxeo-tree-snapshot-core/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.nuxeo.snapshot</groupId>
```

Comment:
Use `org.nuxeo.ecm.platform` here, like most addons, we don't want to multiply the group ids and especially have them be specific to the addon.